tests/testthat/test-encrypt-wrapper.R

```diff
@@ -30,10 +30,12 @@ test_that("nse 2", {
   filename <- tempfile()
   on.exit(file_remove_if_exists(filename))
 
-  encrypt(write.csv(iris, filename, row.names = FALSE), x)
+  d <- data.frame(a = 1:10, b = sample(1:10))
```

Comment:
Why can't it work with iris by changing some arguments? Sorry if I'm missing something obvious.

Reply:
The message from CRAN suggests adding `stringsAsFactors = TRUE` to `read.csv` and friends to preserve behaviour but only if the order of factors never matters and to make sure tests pass with both `stringsAsFactors` defaults.  I could do that but `iris` *does* have a factor for the `Species` column, so I would have ended up making the test more complicated and fragile.

The aim of this test is to try and roundtrip a csv - no need to get concerned with factors, so I am now just creating a small example with no factor problems.

Reply:
Ok, it makes sense, thank you!